changelog.yaml

```diff
@@ -14,9 +14,25 @@
 
 #      jiraIssueNumber: <Internal Jira issue number>
 
-- version: "NEXT"
-  date: TBD
-  
+- version: "0.6.0"

```

Comment:
The version must not be changed in the PR, but during the release. Only add the `change` in `changes`.

Reply:
Done